docs/commands.rst

```diff
@@ -297,55 +304,40 @@ Examples of patterns are available in ``tests/OK/patterns.lp``.
 The ``set`` command is used to control the behaviour of Lambdapi and
 extension points in its syntax.
 
-**declared identifiers** The following code declares a new valid identifier.
-
-::
-
-   set declared "ℕ"
-   set declared "α"
-   set declared "β"
-   set declared "γ"
-   set declared "x₁"
-   set declared "x₂"
-   set declared "x₃"
-
 **prefix symbols** The following code defines a prefix symbol for
 negation.
 
 ::
 
-   set prefix 5 "¬" ≔ neg
+   set prefix 5 "¬" ≔ neg;
 
 **infix symbols** The following code defines infix symbols for addition
 and multiplication. Both are associative to the left, and they have
 priority levels ``6`` and ``7`` respectively.
 
 ::
 
-   set infix left 6 "+" ≔ add
-   set infix left 7 "×" ≔ mul
+   set infix left 6 "+" ≔ add;
+   set infix left 7 "×" ≔ mul;
 
 The modifier ``infix``, ``infix right`` and ``infix left`` can be used
 to specify whether the defined symbol is non-associative, associative to
 the right, or associative to the left. The priority levels are floating
 point numbers, hence a priority can (almost) always be inserted between
 two different levels.
 
-*WARNING:* some checks are performed upon the declaration of infix
-symbols and identifiers, but they are by no means sufficient (it is
-still possible to break the parser by defining well-chosen tokens).
-
-**quantifier symbols** The representation of a symbol can be modified to
-make it look like a usual quantifier (such as ``∀``, ``∃`` or ``λ``).
-Symbols declared as quantifiers can be input using a “quantifier” syntax
-and their printing is changed:
+**quantifier symbols** Any symbol can be input as a quantifier (as done usually
+with symbols such as ``∀``, ``∃`` or ``λ``), provided that it
+is prefixed with a backquote `` \` ``. However, such terms will be printed as
+quantifiers only if they are declared so using the command ``set quantifier``:
 
 ::
 
-   set quantifier ∀ // : Π {a}, (T a → Prop) → Prop
-   compute ∀ {a'} (λx:T a,p) // prints ∀x:T a,p
-   compute ∀ (λx:T a, p) // prints ∀x,p
-   type ∀x,p // quantifiers can be written as such
+   set quantifier ∀; // : Π {a}, (T a → Prop) → Prop
+   compute ∀ {a'} (λx:T a,p); // prints `∀x:T a,p
+   compute ∀ (λx:T a, p); // prints `∀x,p
```

Comment:
space missing after \forall

Reply:
btw, do you print a space after \lambda, \Pi and quantifiers in print.ml ?

Reply:
Aaah well seen, I'd have forgotten that.